lldb/source/Target/Statistics.cpp

```diff
@@ -408,3 +410,21 @@ llvm::json::Value DebuggerStats::ReportStatistics(
 
   return std::move(global_stats);
 }
+
+llvm::json::Value SummaryStatistics::ToJSON() const {
+  return json::Object{{
+      {"name", GetName()},
+      {"type", GetSummaryKindName()},
+      {"invocationCount", GetSummaryCount()},
```

Comment:
changet to `"count"`? `"invocationCount"` is not really telling us anything more that `"count"` would? Please let me know if anyone disagrees.

Reply:
Will fix, I updated the variable but not the json, facepalm